src/algorithm/MEMCPY-Cuda.cpp

```diff
@@ -157,7 +157,7 @@ void MEMCPY::runCudaVariant(VariantID vid, size_t tune_idx)
         run_params.validGPUBlockSize(block_size)) {
 
       if (tune_idx == t) {
-
+        setBlockSize(block_size);
```

Comment:
@MrBurmark please take a look at these changes and indicate whether you have any questions or concerns.

Reply:
This particular change is to allow capture of the block size as metadata. I'm not thrilled with this but I'm not sure there is a better way to do it either. I have some ideas about refactoring tunings that may improve things a bit but that's longer term.

Reply:
Most kernels' block size are being set by a change in `src/common/GPUUtils.hpp::RAJAPERF_GPU_BLOCK_SIZE_TUNING_DEFINE_BOILERPLATE`. But some like `MEMCPY-Cuda.cpp` don't contain the boilerplate so I had to put them in manually like above. To clarify this was to get per kernel block size and not for the run metadata.